pss/trojan_chunk.go

```diff
@@ -28,173 +28,180 @@ import (
 	"github.com/ethersphere/swarm/storage"
 )
 
-// messageTopic is an alias for a 32 byte fixed-size array which contains an encoding of a message topic
-type messageTopic [32]byte
+// msgTopic is an alias for a 32 byte fixed-size array which contains an encoding of a message topic
+type msgTopic [32]byte
```

Comment:
done.

i had not changed this to `Topic` before since `pss` package _already_ included a `Topic` type. now we have the `trojan` package, we can do it.

in general, i think we should export stuff only when we need to, and this is not used outside of the package for now. 

however, we are close to needing it and i think our path forward is pretty clear for now. so i've exported this as well.